napari/plugins/pypi.py

```diff
@@ -98,7 +108,7 @@ def get_package_versions(name: str) -> List[str]:
 def get_napari_plugin_repos_from_github() -> List[dict]:
     """Return GitHub API hits for repos with "napari plugin" in the README."""
```

Comment:
This doc string is no longer accurate - we now look for the topic not the readme